openapi.yaml

```diff
@@ -329,44 +367,56 @@ paths:
         - $ref: '#/components/parameters/pageNumber'
         - $ref: '#/components/parameters/pageSize'
         - in: query
-          name: filter[book_id]
+          name: filter[bookId]
           description: Search by book ID borrowed.
           required: false
           schema:
             type: string
+            example: '26'
         - in: query
-          name: filter[member_id]
+          name: filter[memberId]
           description: Search by member ID who borrowed the book.
           required: false
           schema:
             type: string
+            example: '001'
         - in: query
-          name: filter[borrow_date]
+          name: filter[borrowDate]
           description: Filter books borrowed on a specific date.
           required: false
           schema:
             type: string
             format: date
+            description: The date in the format YYYY-MM-DD.
+            example: '2023-06-30'
         - in: query
-          name: filter[due_date]
+          name: filter[dueDate]
           description: Filter books due on a specific date.
           required: false
           schema:
             type: string
             format: date
+            description: The date in the format YYYY-MM-DD.
```

Comment:
All descriptions of your date fields are the same. Can you provide more accurate descriptions for them?

Reply:
done